internal/pkg/cli/init.go

```diff
@@ -121,6 +122,7 @@ func NewInitOpts() (*InitOpts, error) {
 	}, nil
 }
 
+// Run executes the bootstrap of a new application.
```

Comment:
nit: maybe `Run executes "project init", "env init", "app init" and "app deploy".`